Sources/OktaIdx/Internal/Implementations/Version1/Utilities/UserAgent.swift

```diff
@@ -1,9 +1,14 @@
-//
-//  UserAgent.swift
-//  okta-idx-ios
-//
-//  Created by Mike Nachbaur on 2021-02-11.
-//
+/*
+ * Copyright (c) 2021, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
```

Comment:
Thanks for catching this, I missed one!

Reply:
Np. I'm used to use the boy scout rule :)